quickstarts/gradle/docker-file-provided-context-dir/build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id 'org.springframework.boot' version '2.5.3'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'org.eclipse.jkube.kubernetes' version '1.5.0-SNAPSHOT'
+    id 'java'
+}
+
+group = 'org.eclipse.jkube.gradle.sample.spring.boot'
+version = '1.5.0-SNAPSHOT'
+sourceCompatibility = '11'
+def contextDirPath = projectDir.getAbsolutePath() + '/src/main/docker-context-dir'
```

Comment:
I guess this is defined here because there's an evaluation problem if inlined in line 31?
Did you try this with a template string and a placeholder?

Reply:
Do you have some example for this?

Reply:
I'll try to change it along with the other things. But you can confirm that this is why you declared the other variable, right?

Reply:
Yes, when I was specifying this directly in configuration. I was seeing problems with `gradle clean build`